src/tests/ll_verification.py

```diff
@@ -8465,6 +8465,153 @@ def ll_ist_cen_bv_03_c(transport, upper_tester, lower_tester, trace):
+    PrimChannelMap  = 0x07  # Advertise on all three channels (#37, #38 and #39)
+    OwnAddrType     = SimpleAddressType.PUBLIC
+    PeerAddrType    = SimpleAddressType.PUBLIC
+    PeerAddress     = toArray(0x456789ABCDEF, 6)
+    FilterPolicy    = AdvertisingFilterPolicy.FILTER_NONE
+    TxPower         = 0
+    PrimAdvPhy      = PhysicalChannel.LE_1M
+    SecAdvMaxSkip   = 0
+    SecAdvPhy       = phy
+    Sid             = 0
+    ScanReqNotifyEnable = 0
+
+    success = preamble_ext_advertising_parameters_set(transport, upperTester, Handle, Properties, PrimMinInterval, PrimMaxInterval,
+                                                      PrimChannelMap, OwnAddrType, PeerAddrType, PeerAddress, FilterPolicy, TxPower,
+                                                      PrimAdvPhy, SecAdvMaxSkip, SecAdvPhy, Sid, ScanReqNotifyEnable, trace)
+    if not success:
+        return success
+
+    advData = [ 0xAA ]
+    success = success and preamble_ext_scan_response_data_set(transport, upperTester, Handle, FragmentOperation.COMPLETE_FRAGMENT, 0, advData, trace)
+    if not success:
+        return False
+
+    success = success and preamble_ext_advertise_enable(transport, upperTester, Advertise.ENABLE, [Handle], [0], [0], trace)
+    if not success:
+        return False
+
+    responses = 0
+
+    for i in range(100):
+        def wait_for_AUX_ADV_IND_end():
+            # Get an ADV_EXT_IND with an aux ptr pointing to an AUX_ADV_IND packet that hasn't been sent yet
+            auxAdvIndPacket = None
+            auxAdvIndEndTs = 0
+            while True:
+                lastPacket = packets.findLast(packet_filter=('ADV_EXT_IND', 'AUX_ADV_IND'))
+                if not auxAdvIndPacket:
+                    auxAdvIndPacket = packets.findLast(packet_filter='AUX_ADV_IND')
+                if lastPacket.type == PacketType.ADV_EXT_IND and auxAdvIndPacket:
+                    # Calculate end of offset window
+                    offsetEnd = (lastPacket.payload['AuxPtr'].auxOffset + 1) * (30 if lastPacket.payload['AuxPtr'].offsetUnits == 0 else 300)
+                    # Expected air time of the AUX_ADV_IND packet (assuming no changes from the last one)
+                    # Note: Packet air length is: pre-amble + AA + header + payload + CRC
+                    airTime = ((2 if auxAdvIndPacket.phy == '2M' else 1) + 4 + 2 + len(auxAdvIndPacket) + 3)*8/(2 if auxAdvIndPacket.phy == '2M' else 1)
+                    # Calculate expected (last possible) end time of the coming AUX_ADV_IND packet
+                    auxAdvIndEndTs = int(lastPacket.ts + offsetEnd + airTime)
+                    break
+                # Don't have the needed packets yet or the last packet was not an ADV_EXT_IND; Wait a little and try again
+                transport.wait(1)
+            if auxAdvIndEndTs + 1 <= transport.get_last_t():
+                # Always wait at least 1 us to avoid deadlocks
+                auxAdvIndEndTs = transport.get_last_t() + 1
```

Comment:
Wouldn't this "+1" and the "+1" in wait_until_t mean that this waits at least 2us?

Reply:
True - I opted for the safe and readable number; anyway, waiting 2 us should be fine as well :-)

Reply:
I am sure it works fine, but the comment is misleading then. Maybe just doing `transport.wait_until_t(max(auxAdvIndEndTs, transport.get_last_t()) + 1)` below would be simpler?

Reply:
Yeah that would be cleaner - I'll change it to use that instead